### PR TITLE
Pub 2221 ET lists

### PR DESCRIPTION
**Pull request by MindGames2015**
**Before creating a pull request make sure that:**

-   [ ] commit messages are meaningful and follow good commit message guidelines
-   [ ] README and other documentation has been updated / added (if needed)
-   [X] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

https://tools.hmcts.net/jira/browse/PUB-2221

### Change description

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ X No
```

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -112,6 +112,7 @@
         "@playwright/browser-chromium": "1.39.0",
         "@playwright/browser-firefox": "1.39.0",
         "@playwright/browser-webkit": "1.39.0",
+        "@types/axios-token-interceptor": "^0.2.7",
```
Comment (KianKwa) [inline package.json:115]: why do u have the axios dependency added for this ticket?

File: src/main/views/et-fortnightly-list.njk [hunk 2]
```diff
@@ -110,7 +110,6 @@
                       <td class="govuk-table__cell">{{ record.respondent }}<br />{{ rep }}: {{ noRep }}</td>
                     {% endif %}
                     <td class="govuk-table__cell">{{ record.hearingType }}</td>
-                    <td class="govuk-table__cell">{{ record.jurisdiction }}</td>
```
Comment (KianKwa) [inline src/main/views/et-fortnightly-list.njk:113]: Nothing wrong with this line here but there is a field called 'jurisdiction' for ET fortnightly list needs removing from the class EtListsService

File: src/test/unit/views/et-daily-list.test.ts [hunk 3]
```diff
@@ -137,18 +137,18 @@ describe('ET Daily List page', () => {
 
     it('should display Case Name without Case Sequence Indicator', () => {
         const cell = htmlRes.getElementsByClassName('govuk-table__cell');
-        const x = 9;
+        const x = 8;
```
Comment (KianKwa) [inline src/test/unit/views/et-daily-list.test.ts:140]: Need to remove CaseType from the test data for unit tests and end2end tests too

**Review by KianKwa — CHANGES_REQUESTED** [review]
Hey Paul I have added a few comments above. Think there are a few more files need changing

File: package.json (hunk 1 above)
Comment (MindGames2015) [reply package.json:115]: I'll remove this line Kian, thanks

File: src/main/views/et-fortnightly-list.njk (hunk 2 above)
Comment (MindGames2015) [reply src/main/views/et-fortnightly-list.njk:113]: Ah missed this thanks kian

File: src/test/unit/views/et-daily-list.test.ts (hunk 3 above)
Comment (MindGames2015) [reply src/test/unit/views/et-daily-list.test.ts:140]: OK, Kian will do

**Review by KianKwa — APPROVED** [review]

**Review by KianKwa — CHANGES_REQUESTED** [review]
Just notice the schema on confluence for ET daily list still have caseType. That needs removing too.

**Review by KianKwa — APPROVED** [review]

**Pull request merged**